streams/src/main/java/org/apache/kafka/streams/processor/internals/StoreChangelogReader.java

```diff
@@ -98,7 +100,7 @@
 
         private long totalRestored;
 
-        // only for active restoring tasks (for standby changelog it is null)
+        // only for active restoring tasks (for standby changelog it is null) WHAT IS NULL?
```

Comment:
This comment is unclear to me, that is why I added the text -- can you explain what it means @guozhangwang ?